packages/ui-library/src/components/modal/index.js

```diff
@@ -3,9 +3,45 @@ import { XIcon } from "@heroicons/react/outline";
 import { forwardRef, Fragment } from "@wordpress/element";
 import classNames from "classnames";
 import PropTypes from "prop-types";
-import Title from "../../elements/title";
 import { useSvgAria } from "../../hooks";
 import { ModalContext, useModalContext } from "./hooks";
+import { classNameMap as titleClassNameMap } from "../../elements/title";
+
+/**
+ * @param {JSX.node} children Title text.
+ * @param {string} [className] Additional class names.
+ * @param {string} [as] Html tag.
```

Comment:
This could also be a `JSX.ElementClass` like the UI library `Title`, right?

Reply:
changed it to `@param {string|JSX.Element} [as="h1"] Base component.`